multi-node/aws/pkg/config/config.go

```diff
@@ -243,6 +242,18 @@ type Config struct {
 	TLSConfig *CompactTLSAssets
 }
 
+//This functionality now requires AWS API access
+//Separated out from cluster.Config() so that we can only compact
+//assets on command=up
+func (c *Config) CompactTLSAssets(tlsConfig *RawTLSAssets) error {
```

Comment:
This originally returned the type rather than assigning to a struct field. Yes it shouldn't be exported.

It might even be better just to remove the function entirely and run

```
compact, err := tlsConfig.Compact(c)
 if err != nil {
 	return fmt.Errorf("failed to compress TLS assets: %v", err)
 }
 c.TLSConfig = compact
```

when you need the TLSConfig. I don't see much value in having a helper function that does that. It's unclear when it should be called.